tfhe/src/integer/gpu/server_key/radix/oprf.rs

```diff
@@ -422,6 +424,149 @@ impl CudaServerKey {
+        streams.synchronize();
+
+        let mut result: CudaUnsignedRadixCiphertext =
+            self.create_trivial_zero_radix(num_blocks_output as usize, streams);
+
+        unsafe {
+            match &self.bootstrapping_key {
+                CudaBootstrappingKey::Classic(d_bsk) => {
+                    cuda_backend_grouped_oprf_custom_range(
+                        streams,
+                        result.as_mut(),
+                        num_blocks_intermediate as u32,
+                        &d_seeded_lwe_input,
+                        decomposed_scalar.as_slice(),
+                        has_at_least_one_set.as_slice(),
+                        num_input_random_bits as u32,
+                        &d_bsk.d_vec,
+                        &self.key_switching_key.d_vec,
+                        d_bsk.input_lwe_dimension,
+                        d_bsk.glwe_dimension,
+                        d_bsk.polynomial_size,
+                        self.key_switching_key.decomposition_level_count(),
+                        self.key_switching_key.decomposition_base_log(),
+                        d_bsk.decomp_level_count,
+                        d_bsk.decomp_base_log,
+                        LweBskGroupingFactor(0),
+                        self.message_modulus,
+                        self.carry_modulus,
+                        PBSType::Classical,
+                        message_bits_count as u32,
+                        post_mul_num_bits as u32,
+                        d_bsk.ms_noise_reduction_configuration.as_ref(),
+                    );
+                }
+                CudaBootstrappingKey::MultiBit(d_bsk) => {
+                    cuda_backend_grouped_oprf_custom_range(
+                        streams,
+                        result.as_mut(),
+                        num_blocks_intermediate as u32,
+                        &d_seeded_lwe_input,
+                        decomposed_scalar.as_slice(),
+                        has_at_least_one_set.as_slice(),
+                        num_input_random_bits as u32,
+                        &d_bsk.d_vec,
+                        &self.key_switching_key.d_vec,
+                        d_bsk.input_lwe_dimension,
+                        d_bsk.glwe_dimension,
+                        d_bsk.polynomial_size,
+                        self.key_switching_key.decomposition_level_count(),
+                        self.key_switching_key.decomposition_base_log(),
+                        d_bsk.decomp_level_count,
+                        d_bsk.decomp_base_log,
+                        d_bsk.grouping_factor,
+                        self.message_modulus,
+                        self.carry_modulus,
+                        PBSType::MultiBit,
+                        message_bits_count as u32,
+                        post_mul_num_bits as u32,
+                        None,
+                    );
+                }
+            }
+        }
+
+        streams.synchronize();
```

Comment:
When you synchronize in the release you don't need to call this streams.synchronize() here  anymore